controller/cartController.js

```diff
@@ -0,0 +1,22 @@
+import { cartService } from '../service';
+
+const updateCartItemQuantity = async (req, res) => {
+  try {
+    const { userId, quantity, isTrue } = req.body;
```

Comment:
```suggestion
    const { userId, quantity, isTrue } = req.body;
```
`userId`는 body에서 받아오시면 안되고, req 헤더에 들어있는 user에서 꺼내와야 합니다.
토큰 체크하는 middleware를 통과시키면, 리퀘스트에 user 정보 들어가있죠 지현님?!

Reply:
네 미들웨어 통과 시키게 되면, 유저 정보가 잘 받아와집니다. 확인 완료!